docker-compose.yml

```diff
@@ -17,6 +17,8 @@ services:
       - cluster.name=docker-cluster
       - bootstrap.memory_lock=true
       - "ES_JAVA_OPTS=-Xmx512m -Xms512m"
+    networks:
```

Comment:
I guess this network is for varnish, but we can't enable it by default. I've seen that you created a tutorial for that which is really cool. So please remove all of the references form our docker-compose files and keep only the docker files and docker-compose.varnish.